### DISCUSSION TITLE
index.html not updating to reflect publication.yml and config.yml changes

### DISCUSSION THREAD

**Discussion by sophielamb** [thegetty/quire#107] [Get Help] [answered]
I have been updating publication.yml and config.yml with our project's data. I've discovered, on inspecting the browser preview, that the index.html has not been updated to receive these changes ie. the head `<meta>` tags and the body `<span>` tags still contains the Dorothea Lange / Walker Evans template code. Although my preview looks fine, I'm not sure how or why the html meta tags are unaffected.

Can I dive into the index.html file and change the meta tags and some of the span tags? Or is it better practice to control changes purely through mark-down and yaml files? If so, why wouldn't this be reflected in the html? 

Thanks 
Sophie

**Comment by geealbers** [comment, marked as answer]
Hi @sophielamb, I take it you're looking at the `index.html` file inside the `site` directory of your project? That file won't actually update until you run `quire site`. `quire preview` builds the site in the background and, somewhat non-intuitively, doesn't update the `site` files.

**Reply by sophielamb** [reply]
Hi @geealbers thanks for replying. 
That could be it. Considering it's not causing any visible problems, I'm going to assume they'll resolve as all of the content is changed in due course.
